firebase-dynamic-links/src/main/java/com/google/firebase/dynamiclinks/DynamicLink.java

```diff
@@ -383,7 +379,7 @@ public Builder(@NonNull String packageName) {
         return this;
       }
 
-      /** @return the iPad parameters ID of the app. */
+      /** @return the link to open on Android if the app isn't installed. */
```

Comment:
that is a drastic change from iPad to Android, pls make sure it's intended

Reply:
Yeah, it's quite a change. If you see the code, the `getParcelable` is reading `KEY_ANDROID_FALLBACK_LINK` :-)